domains/Qwertx.is-an.app.json

```
@@ -0,0 +1,14 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+  "description": "My personal site and other stuff",
+  "domain": "is-an.app",
+  "subdomain": "qwertx",
+  "owner": {
+    "repo": "https://github.com/notqwertz/free-domains",
```

Is that repo with content for the subdomain?

```suggestion
    "repo": "https://github.com/NotQWertz/NotQWertz.github.io",
```